docs/examples/segmentedcontrol/defaultExample.js

```diff
@@ -0,0 +1,27 @@
+// @flow strict
+import { type Node, useState } from 'react';
+import { Box, Text, SegmentedControl, Icon } from 'gestalt';
+
+export default function Example(): Node {
+  const [itemIndex, setItemIndex] = useState(0);
+  const items = [
```

Comment:
nit: consts like these should be defined outside the component, so they're not recreated on every render (this doesn't matter here, but we want to make sure the examples we show are production-ready code)